m1.lesson1.web_calculator/basicStyle.css

```diff
@@ -0,0 +1,62 @@
+@charset "UTF-8";

```

Comment:
not require when not using non-ascii chars

Reply:
also, the 2 top elements not aligned
![image](https://user-images.githubusercontent.com/11487010/38465368-74378dba-3b23-11e8-9c34-4802eb82021c.png)